about.html

```diff
@@ -0,0 +1,135 @@
+                <a href="./index.html"><i class="fa fa-home"></i></a> > <a href="#">about </a> 
+            </div>
+            <div class="intro">
+                <h1>
+                    Artificial intelligence,<br>are we talking?
+                </h1>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.<br>
+                    Animi numquam ipsa quis deleniti quia laudantium voluptatem perferendis, saepe cum aliquid optio unde!<br>A saepe pariatur commodi et
+                    blanditiis,<br>
+                    neque nemo, neque nemo!
+                    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Animi numquam ipsa quis deleniti quia<br>
+                    laudantium voluptatem perferendis, saepe cum aliquid optio unde! A saepe pariatur commodi et
+                    blanditiis,<br>
+                    neque nemo, neque nemo!<br>
+                    laudantium voluptatem perferendis, saepe cum aliquid optio unde! A saepe pariatur commodi et
+                </p>
+                <p>Please contact us per Email for any further question<br>
+                    about Haiti Tech Summit 2023<br>
+                    <a mailto='htsummit2023@hts.org'>htsummit2023@hts.org</a></p>
+            </div>
+        </section>
+
+        <section class="program">
+            <h2>Haiti tech summit 2023 Logo</h2>
+            <hr>
+            <p>The logo of Haiti Tech Summit 2023 was decided through the logo competition from 8, June to 7, July.</p>
+            <img src="./public/images/logo.jpg" alt="">
+        </section>
+
+        <section class="speakers">
+            <h2>See the past Haiti tech summits</h2>
+            <hr>
+            <p>Take a look at the last two Haiti tech summit which took place in Haiti.</p>
+            <div class="cards">
+                <div class="card">
+
+                    <div class="front">
+                        <h2>2020</h2>
+                        <p>Haiti tech summit 2020</p>
+                    </div>
+                </div>
+                <div class="card">
+
+                    <div class="front">
+                        <h2>2022</h2>
+                        <p>Haiti tech summit 2022</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="partner">
+            <h2>Partner</h2>
+            <hr>
+            <div class="cards">
+                <img src="./public/images/logo_airbnb.png" alt="Logo Airbnb">
+                <img src="./public/images/logo_nestle.png" alt="Logo Nestle">
+                <img src="./public/images/logo_mozilla.png" alt="Logo Mozilla">
+                <img src="./public/images/logo_blender.png" alt="Logo Bendler">
+            </div>
+        </section>
+
+        <section class="footer">
+            <div class="cards">
+                <img src="./public/images/creative_commons.png" alt="Logo Creative Commons">
```

Comment:
- [ Required ] Please consider `customizing` (use your logo) and making the `size` of the logo in the footer consistent with the size of the logo in the Behance design. Currently, the size of the logo appears to be wide and stretched compared to the design on Behance. Please see the screenshot below and rectify this.

## Behance design
![Screenshot from 2023-04-27 00-35-33](https://user-images.githubusercontent.com/16392046/234708538-371e8e32-fe43-42c5-a4a3-44c1b1140cc8.png)


## Your design
![Screenshot from 2023-04-27 00-33-42](https://user-images.githubusercontent.com/16392046/234708124-da6db07c-5c95-43af-9512-f593b144c897.png)